### PR TITLE
Doc: more details on hotfix procedure

### PR DESCRIPTION
**Pull request by mbounabi**

### CODE REVIEW

File: readme_release.md [hunk 1]
```diff
@@ -19,20 +19,36 @@ Then follow the instructions given by the script, and also:
...
+* Cherry-pick the commits you want to release:
+	```
+	git cherry-pick <commit_id> # Each commit_id of the hotfix PR
+	```
+* Merge the content of the new release branch with the hotfix commits to the 'release' branch:
+	```
+	git checkout release		
+	git merge --no-ff <release_x.yy.z>
+	```
+* Tag the new release:
```
Comment (pbougue) [inline readme_release.md:44]: minor: Not sure I remember correctly, but I think at this step you also have to populate PR name and link properly, with enough blank lines to have a clean message like here : https://github.com/CanalTP/navitia/releases/tag/v2.58.0

**Review by pbougue — APPROVED** [review]
Nice, thanks for this add!

File: readme_release.md (hunk 1 above)
Comment (mbounabi) [reply readme_release.md:44]: Enough being how many? Looks like I missed this part and every other hotfix releases. For ex:
https://github.com/CanalTP/navitia/releases/tag/v2.52.1

Comment (pbougue) [reply readme_release.md:44]: Looks like it's one fully blank line after title:
using `git show v2.57.0` (which looks OK on github)
```bash
Version 2.57.0

  * Kraken: Add ptref shortcut between physical_mode and jpps  <https://github.com/CanalTP/navitia/pull/2417>
```

`git show v2.58.1` (less OK)
```bash
v2.58.1: Ed: add rtree to speedup stop_point association to admins <https://github.com/CanalTP/navitia/pull/2466>
```

**Review by benoit-bst — APPROVED** [review]

**Pull request merged**